### DISCUSSION TITLE
Bounding boxes and debugging

### DISCUSSION THREAD

**Discussion by sygys** [blakeblackshear/frigate#14152] [Ask A Question] [answered]
I updated to version 0.14.x and am using the frigate add-on in home assistant. Before this major overhaul i was able to turn on bounding  boxes and see zones. In this last version all these features are now gone? Atleast i cant find these toggles anymore in the new ui.

It would be great to get them back though. because for debugging boundingbox sizes etc. its pretty handy to exctually see what is happening.

**Comment by NickM-27** [comment]
The debug view is in settings and offers all the same options

**Comment by sygys** [comment]
Maybe it's me but in the addon on home assistant I can't find any settings to let it show bounding boxes. Not in the overall settings and also not in the camera view settings. But it could be that I need to update the hacs integration. I see that it has an update

**Reply by hawkeye217** [reply]
![Screenshot 2024-10-04 at 9 19 08 AM](https://github.com/user-attachments/assets/ca67ba18-be22-4249-a208-7eb50244e78b)

**Comment by sygys** [comment]
It seems I don't have a debug button on my mobile phone. Or does this only work on pc? 
![Screenshot_20241004_161959_Home Assistant](https://github.com/user-attachments/assets/2e011d6b-c7aa-49fb-b2ad-9ffb16b69dad)

**Reply by NickM-27** [reply, marked as answer]
Press the settings cog and then press settings

**Comment by sygys** [comment]
Damn thanks.. I missed the upper part where you can choose different tabs.... Sorry my bad
![Screenshot_20241004_162321_Home Assistant](https://github.com/user-attachments/assets/114589c8-149d-4b22-baa1-44948fedbd4a)
